mentors/cpp/d.json

```
@@ -1 +1,8 @@
-[]
+[{
+    "github_username": "drentropy",
+    "name": "Ronald Legere",
+    "link_text": "",
+    "link_url": "",
+    "avatar_url": null,
+    "bio": "C++ is not elegant but it does provide awesome power and expressiveness when mastered. These days I primarily use C++ for audio development but also tinker with gaming."
+  }]
```

```suggestion
[
  {
    "github_username": "drentropy",
    "name": "Ronald Legere",
    "link_text": "",
    "link_url": "",
    "avatar_url": null,
    "bio": "C++ is not elegant but it does provide awesome power and expressiveness when mastered. These days I primarily use C++ for audio development but also tinker with gaming."
  }
]
```
Content is fine, just minor formatting changes.

---

Thanks! I didnt notice that for some reason.